Makefile

```diff
@@ -2,10 +2,13 @@ ZPOOL=""
 SERVER=""
 SVN_REL_URL=https://svn.freebsd.org/base/releng/11.1
 
-install:
+/usr/src/cddl:
+	( cd /usr/src && svnlite checkout $(SVN_REL_URL)/cddl/ )
+/usr/src/sys/cddl:
+	( cd /usr/src && mkdir -p sys && cd sys && \
```

Comment:
Can be simplified to `mkdir -p /usr/src/sys && cd /usr/src/sys`

Reply:
thank you, @gronke.
it's been a long work-week.

Reply:
I had to think twice before posting this! I wasn't sure if it really was an improvement. My final decision to comment was because it reduces complexity, which is always good for readability